server/events.py

```diff
@@ -23,6 +23,7 @@ class EventType(str, Enum):
     MOVE = "move"
     REPLACE = "replace"
     ERROR = "error"
+    SEND_BUGS = "bugs"
```

Comment:
Having this type of event means that the client is going to request the bugs, how are we going to handle this?

We still haven't discussed in what cases the bugs should be broadcasted. Does this happen at a particular event? Does this happen on a timer?

An issue for this could be opened.

Reply:
I was thinking that the frontend would keep a timer going and every x seconds, dependent on difficulty, they could request the backend to send buggy code

Reply:
That seems good. However (this might be a very stupid question) don't we risk that the user could change the timer because of it being on the front end?

Reply:
I had thought about this, actually. I guess it could be possible to mess with the timer. I had thought maybe it could be invisible to the front end but then again, it would be nice to see how long it takes for someone to fix code (a challenge or pre-determined code). At the same time, if they're going to change the timer on the frontend, then there is no point in them playing the game 😆 

We could perhaps implement it on the backend instead, if that seems more logical?

Reply:
I think we could do that, it shouldn't be too big of a problem I think? The code is asynchronous after all.

I'm opening an issue for this.